qemu/tests/cfg/netkvm_in_use.cfg

```diff
@@ -10,7 +10,7 @@
     driver_name = "netkvm"
     run_bgstress = netperf_stress
     bg_stress_run_flag = netperf_run
-    netperf_link = netperf-2.6.0.tar.bz2
+    netperf_link = netperf-2.7.1.tar.bz2
```

Comment:
It seems that windows only has `deps/netperf/netperf.exe` (2.6.0), I am not sure 2.7.1 can work with it together, please help confirm this one.

Reply:
@PaulYuuu  It test pass.